core/src/main/java/feast/core/job/dataflow/DataflowJobManager.java

```diff
@@ -262,6 +286,7 @@ private ImportOptions getPipelineOptions(
     pipelineOptions.setUpdate(update);
     pipelineOptions.setRunner(DataflowRunner.class);
     pipelineOptions.setJobName(jobName);
+    pipelineOptions.setLabels(labels);
```

Comment:
Cmiiw, but this overwrites all labels that could be set in application properties. Shouldn't we extend them instead?

Reply:
Yes I think we should extend them. Thanks for checking